From 01f11248e5e70a0f93a73190ef5505365c64b773 Mon Sep 17 00:00:00 2001
From: Robby Cai <r63905@freescale.com>
Date: Mon, 4 May 2009 10:51:07 +0800
Subject: [PATCH 055/280] ENGR00112131 Fix compiler error due to undefined
 regs on MX51

Fix compiler error of vpu lib due to undefined registers on MX51

Signed-off-by: Robby Cai <r63905@freescale.com>
---
 vpu/vpu_io.c | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/vpu/vpu_io.c b/vpu/vpu_io.c
index 93bc9f5..05dece9 100644
--- a/vpu/vpu_io.c
+++ b/vpu/vpu_io.c
@@ -78,10 +78,10 @@ Uint32 use_iram_table[][NR_ENTRY] = {
 	 *
 	 * {VPU Register Address Offset, IRAM Size, Flag indicating used or not}
 	 */
-	{CMD_SET_FRAME_AXI_DBK_ADDR, 25600, 0},
-	{CMD_SET_FRAME_AXI_OVL_ADDR, 8000, 0},
-	{CMD_SET_FRAME_AXI_IP_ADDR, 6400, 0},
-	{CMD_SET_FRAME_AXI_BIT_ADDR, 6400, 0},
+	{0x19C, 25600, 0},
+	{0x1A0, 8000, 0},
+	{0x198, 6400, 0},
+	{0x190, 6400, 0},
 };
 
 int get_iram_setting(struct iram_t iram, Uint32 array[][NR_ENTRY],
-- 
1.8.0

